pkg/kubectl/cmd/get_test.go

```diff
@@ -740,10 +779,10 @@ func TestWatchSelector(t *testing.T) {
 	cmd.Flags().Set("selector", "a=b")
 	cmd.Run(cmd, []string{"pods"})
 
-	expected := []runtime.Object{&api.PodList{Items: pods}, events[0].Object, events[1].Object}
+	expected := []runtime.Object{podList, events[2].Object, events[3].Object}
 	actual := tf.Printer.(*testPrinter).Objects
 	if !reflect.DeepEqual(expected, actual) {
-		t.Errorf("unexpected object:\nExpected: %#v\n\nGot: %#v\n\n", expected[0], actual[0])
+		t.Errorf("unexpected object:\nExpected: %#v\n\nGot: %#v\n\n", expected, actual)
```

Comment:
This actually just changes this case to match the others (see  ```TestWatchResource``` and ```TestWatchResourceIdentifiedByFile```). The information it conveys is basically a mismatch in the number lens of actual vs expected.  Dumping the actual structures not very useful either as the huge output is not diffable with tools.